AAUnlimited/MemMods/AAPlay/Events/Loads.cpp

```diff
@@ -405,6 +406,12 @@ void __stdcall SaveLoadEvent() {
 void __stdcall TransferInEvent(ExtClass::CharacterStruct* character, wchar_t* fileName) {
 
 	AAPlay::InitTransferedCharacter(character);
+	std::string path = General::CastToString(fileName);
+	std::size_t found = path.rfind("male\\");
+	if (found != std::string::npos)
```

Comment:
a code style change:
either put everything on the same line or add the {} even if it's just one statement